README.md

```diff
@@ -21,13 +21,19 @@ plugins:
 ```yaml
 custom:
   globalTables:
+    version: v1 # optional, please use 'v1' for 2017.11.29 or 'v2' for 2019.11.21 version creation
```

Comment:
@Alexandelphi  Can you please mention that v1 is default..

Reply:
@rrahul963 just updated and added the updated version with
"version: v1 # optional, default is 'v1' (2017.11.29), please use 'v2' for (2019.11.21) version creation"